chainercv/visualizations/vis_point.py

```diff
@@ -4,32 +4,32 @@
 from chainercv.visualizations.vis_image import vis_image
 
 
-def vis_keypoint(img, keypoint, kp_mask=None, ax=None):
-    """Visualize keypoints in an image.
+def vis_point(img, point, mask=None, ax=None):
+    """Visualize points in an image.
 
     Example:
 
         >>> import chainercv
         >>> import matplotlib.pyplot as plot
         >>> dataset = chainercv.datasets.CUBKeypointDataset()
-        >>> img, keypoint, kp_mask = dataset[0]
-        >>> chainercv.visualizations.vis_keypoint(img, keypoint, kp_mask)
+        >>> img, point, mask = dataset[0]
+        >>> chainercv.visualizations.vis_point(img, point, mask)
         >>> plot.show()
 
     Args:
         img (~numpy.ndarray): An image of shape :math:`(3, height, width)`.
             This is in RGB format and the range of its value is
             :math:`[0, 255]`. This should be visualizable using
             :obj:`matplotlib.pyplot.imshow(img)`
-        keypoint (~numpy.ndarray): An array with keypoint pairs whose shape is
-            :math:`(K, 2)`, where :math:`K` is
-            the number of keypoints in the array.
+        point (~numpy.ndarray): An array of point coordinates whose shape is
+            :math:`(P, 2)`, where :math:`P` is
+            the number of points.
             The second axis corresponds to :math:`y` and :math:`x` coordinates
-            of the keypoint.
-        kp_mask (~numpy.ndarray, optional): A boolean array whose shape is
-            :math:`(K,)`. If :math:`i` th index is :obj:`True`, the
-            :math:`i` th keypoint is not displayed. If not specified,
-            all keypoints in :obj:`keypoint` will be displayed.
+            of the points.
+        mask (~numpy.ndarray, optional): A boolean array whose shape is
```

Comment:
`optional` in the type description may be against to our convention.
c.f. https://github.com/chainer/chainercv/blob/master/chainercv/visualizations/vis_bbox.py#L27-L29